vllm/lora/layers.py

```diff
@@ -1068,6 +1069,8 @@ def _get_logits(
         ).index_select(0, indices_padded).nan_to_num_(nan=float("-inf"),
                                                       posinf=float("inf"),
                                                       neginf=float("-inf")))
+        if current_platform.is_hpu():
+            lora_logits = lora_logits[:logits.shape[0], :]
```

Comment:
As discussed on Slack,we should add comments about this

Reply:
Done